CONTRIBUTING.rst

```diff
@@ -85,10 +85,14 @@ Ready to contribute? Here's how to set up `spidermon` for local development.
     $ pip install -r requirements-docs.txt
     $ tox
 
+#. Install the git hook scripts from pre-commit::
+
+    $ pre-commit install
+
 #. Make sure that your code is correctly formatted using `black`_ . No code will
    be merged without this step::
```

Comment:
I think this step is no longer needed, pre-commit hook should be handling it

Reply:
Afaik, pre-commit only check that it was done before allowing the commit. Developers are still responsible for doing it themselves.

Reply:
Usually we configure pre-commit to actually change the files. So, it runs black and other tools; if there are no changes, it allows to commit, but if there are changes, then it blocks the commit - but the files are changed on disk. Then, user needs to add the changed files to the git staging area, and then commit again. 

This workflow is nice, because there is no opportunity e.g. to use a different black version, there is no need to run formatters manually, but also there is a change to review the changes before making a commit.

Reply:
You are right. This part is now removed.